include/amici/solver.h

```diff
@@ -75,7 +75,7 @@ class Solver {
      * @param model pointer to the model object
      */
 
-    void setupAMIB(BackwardProblem *bwd, Model *model);
+    void setupB(BackwardProblem *bwd, Model *model);
```

Comment:
why shorter? + maybe remove the empty line before?

Reply:
AMI prefix is a relict from C times. We have amici namespace now.